portal/templates/grade/gradebook.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'layouts/base.html' %}
+
+{% block title %}Gradebook{% endblock %}
```

Comment:
The grade directory should be inside of the teacher directory in order to conform to the new project structure